### PR TITLE
Bundle update all (and fix repeat sqlite3 reference)

### PR DESCRIPTION
**Pull request by remi**
Bundle update all steps and fix "Your Gemfile lists the gem sqlite3 (>= 0) more than once" warning

Rails 4.2.6

Update to previous unmerged #24

### CODE REVIEW

File: 6-task-queueing/Gemfile.lock [hunk 1]
```diff
@@ -156,157 +173,165 @@ GEM
...
+    foreman (0.81.0)
       thor (~> 0.19.1)
     formatador (0.2.5)
-    gcloud (0.4.0)
-      beefcake (~> 1.0)
-      digest-crc (~> 0.4)
-      google-api-client (~> 0.8.3)
-      mime-types (~> 2.4)
-      zonefile (~> 1.04)
+    gcloud (0.0.7)
```
Comment (jeffmendoza) [inline 6-task-queueing/Gemfile.lock]: wrong gcloud version here

File: 7-compute-engine/Gemfile.lock [hunk 2]
```diff
@@ -156,171 +173,183 @@ GEM
...
     fog-xml (0.1.2)
       fog-core
       nokogiri (~> 1.5, >= 1.5.11)
     formatador (0.2.5)
-    gcloud (0.2.0)
-      beefcake (~> 1.0)
-      digest-crc (~> 0.4)
-      google-api-client (~> 0.8.3)
-      mime-types (~> 2.4)
+    gcloud (0.0.7)
```
Comment (jeffmendoza) [inline 7-compute-engine/Gemfile.lock]: same here

**Comment by remi** [comment]
ACK

**Comment by jeffmendoza** [comment]
LGTM with gcloud <-> gapiclient fixed

**Pull request merged**